Provider/src/test/java/com/spotify/confidence/ConfidenceFeatureProviderTests.kt

```diff
@@ -587,6 +587,7 @@ internal class ConfidenceFeatureProviderTests {
         )
```

Comment:
We need tests for the `contextChanges` behaviour.

Reply:
[done](https://github.com/spotify/confidence-openfeature-provider-kotlin/pull/133/commits/b445658db9247d6a7452c167d41ecd446622a8d4)